packages/graph/hash_graph/tests/integration/postgres/links.rs

```diff
@@ -31,16 +31,20 @@ async fn insert() {
         1,
     );
 
-    let person_a_identifier = api
+    let person_a_metadata = api
         .create_entity(person_a, person_type_id.clone(), None)
         .await
         .expect("could not create entity");
 
-    let person_b_identifier = api
+    let person_a_identifier = person_a_metadata.identifier();
+
+    let person_b_metadata = api
         .create_entity(person_b, person_type_id.clone(), None)
         .await
         .expect("could not create entity");
 
+    let person_b_identifier = person_b_metadata.identifier();
+
     api.create_link(
         person_a_identifier.entity_id(),
```

Comment:
Rather than the intermediary identifier above can't we just call `person_a_metadata.identifier().entity_id()` here?

Reply:
Can do that, but isn't `person_a_identifier` and `person_b_identifier` used twice? Therefore there'd be duplicate calls to `identifier()` which seems like a worse pattern, but perhaps I'm wrong

Reply:
It's really not too important and shouldn't block merging. I just think these tests are already very verbose, and long, and the variable names are pretty large. Especially having variable names that are so similar (`person_b_metadata` vs `person_b_identifier`) makes it worse to read as well as write imo.

I think shortening it by calling `identifier()` twice is fine and will actually help a fair bit.

Reply:
There are 30 additions in this file alone

Reply:
Have removed them in https://github.com/hashintel/hash/pull/1199/commits/7d985846626654fe3b6f0f5da545a963c394aeae

Reply:
Thanks!